docs/development/new-package-manager-template.md

```diff
@@ -0,0 +1,15 @@
+<!-- Please copy/paste this template and fill it in on request from the Renovate maintainers. -->
+
+**What package manager do you want Renovate to support?**
```

Comment:
Do we want to know if the manager is in wide use or more niche? I sometimes see feature requests to support a quite obscure package manager. Do we intend to support all package managers ever, or do we (want to) have criteria for inclusion?

The reason I'm asking about the criteria is that we incur a support burden for each feature we add, and we will have maintain both code and documentation for each feature. Maybe it might be good to have a better view of "maintainer time vs amount of people we make happy with the feature"?

Let me know your thoughts on this, as I'm curious how you feel about all this. 😉

Reply:
The burden for managers is not too bad, however it would be useful to know how popular it is.